tests/__main__.py

```diff
@@ -0,0 +1,175 @@
+import unittest
+
+from pgmap.io import barcode_reader, fastx_reader, library_reader
+from pgmap.trimming import read_trimmer
+from pgmap.alignment import pairwise_aligner
+from pgmap import counter
+from pgmap.model.paired_read import PairedRead
+
+
+class TestPgmap(unittest.TestCase):
+
+    def test_read_fastq(self):
+        count = 0
+
+        for sequence in fastx_reader.read_fastq("example-data/three-read-strategy/HeLa/PP_pgRNA_HeLa_S1_I1_001_Sampled10k.fastq.gz"):
```

Comment:
Just saw this is a copy paste error. Should test on a non .gz file. Will fix in a later commit